src/main/java/com/cloudlabs/server/compute/ComputeService.java

```diff
@@ -5,8 +5,5 @@
 import java.util.concurrent.TimeoutException;
 
 public interface ComputeService {
-    boolean createPublicInstance(Compute computeInstanceMetadata) throws IOException, InterruptedException, ExecutionException, TimeoutException;
-    String reserveStaticExternalIPAddress(String project, String region, String ipAddressName);
-    String getExternalStaticIPAdress(String project, String region, String addressName);
-    String assignStaticExternalIPAddress(String project, String zone, String instanceName, String ipAddress, String networkInterfaceName) throws IOException;
-}
+    boolean deleteInstance(String project, String zone, String instanceName) throws IOException, InterruptedException, ExecutionException, TimeoutException;
```

Comment:
Lack of function implementation, see below:
https://cloud.google.com/compute/docs/instances/deleting-instance#java